docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -380,6 +384,12 @@ variables; worker nodes **must** specify the `CUBESTORE_WORKER_PORT` and
 `CUBESTORE_META_ADDR` environment variables. More information about these
 variables can be found [in the Environment Variables reference][ref-config-env].
 
+<!-- prettier-ignore-start -->
+[[info | ]]
+| To fully take advantage of the worker nodes in the cluster, we **strongly**
```

Comment:
Maybe it's too late, but ain't it enough to say something "strongly" in the info box without using **bold font**? 😀

Reply:
I've usually bolded "strongly" in the docs wherever we recommend something like that because it draws attention